src/reference/modules/installation/pages/upgrade/notes.adoc

```diff
@@ -38,55 +45,52 @@ Two cases are unsupported:
 
 === HTTPS certificate
 
+[[_changed_default_certificate_upgrade]]
 ==== Changed default certificate
 
-The certificate used by default for HTTPS access is now the same as was already used for policy
-updates. This means you will see a warning message indicating the server certificate has changed
-(if you used Rudder's self-signed certificate). This is expected, and you can accept to update the
-known certificate.
+The certificate used for HTTPS access is now the same as the one used for policy updates. During the upgrade from 6.x, the HTTPS certificate
+is replaced by the certificate used by policy update.
+After the upgrade from 6.x, you will see a warning message indicating the server certificate has changed. This is expected, and you can accept to update the
+certificate if you used Rudder's self-signed certificate.
 
-==== Use a custom certificate
-
-===== Server side
+==== Server side - Use a custom certificate
 
 If you used a custom certificate, you will need to apply manual changes to Rudder's configuration.
 As the self-signed certificate is used for internal communication, you will need to create a
-dedicated virtual host for the Web and API access.
+dedicated virtual host for the Web and API access. You need a way to distinguish access to these two virtual
+hosts, most often by IP, Port or ServerName (using specific DNS name).
 
 The easiest way is probably to start over from the new default configuration and adapt it to your
-needs (you can access it as a new configuration file deployed by your package manager or directly
+needs. You can access it as a new configuration file deployed by your package manager or directly
 in https://raw.githubusercontent.com/Normation/rudder/master/relay/sources/apache/rudder-vhost.conf[our repository].
 
-===== Agent side - Server certificate verification
+You should only edit the virtual hosts in `/etc/apache2/sites-available` or `/etc/httpd/conf.d` which
+is treated as a configuration file by package managers and not automatically replaced. The most common approach is to
+comment the first virtual host and uncomment the last two virtual hosts, and distinguish either by hostname, IP or by port the virtual host
+for policy distribution and the virtual host for Web Interface/API
+
+In addition, we also redirect all requests to port 80 to 443 by default.
```

Comment:
```suggestion

```

this does not match the title of the section (and is probably not really relevant anyway)